src/scribe-plugin-content-cleaner.js

```diff
@@ -4,21 +4,15 @@ module.exports = function(scribe) {
     var COMMAND_NAME = "cleanup";
 
     var filters = [
-        (text) => {
-                return text.replace(/(&nbsp;)/gi, " ");
-        },
-        (text) => {
-            return text.replace(/\s+/g, " ");
-        },
-        (text) => {
-            return text.replace(/ -- /g, " &mdash; ");
-        },
-        (text) => {
-            return text.replace(/(<br\s*\/?>){3,}/gi, '<br>');
-        },
-        (text) => {
-            return text.replace(/<p>\s*?<br\s*\/?><\/p>/g, '');
-        }
+        (text) => text.replace(/(&nbsp;)/gi, " "),
+        (text) => text.replace(/\s+/g, " "),
+        (text) => text.replace(/ -- /g, " &ndash; "),
```

Comment:
Are spaces around the dash definitely [correct](http://www.thepunctuationguide.com/en-dash.html)?